Makefile

```diff
@@ -24,25 +24,27 @@ docker-run-currentweather: ## Running your custom-built docker image locally
 docker-push: ## Pushing the freshly built image to the registry
 	docker push $(DOCKER_USERNAME)/currentweather-nodejs
 
-docker-clean: ## Remove the stuff we built locally afterwards
```

Comment:
My naming sheme is <tool> – <command>. docker-stop would imply that `docker stop` is called